packages/unminify/src/transformations/__tests__/un-jsx.spec.ts

```diff
@@ -201,11 +199,68 @@ const Foo = () => {
 const Foo = () => {
   const Component = g ? "p" : "div";
   const Component_1 = r ? "a" : "div";
-  return <div><Component_1 key="b">bar</Component_1><Component key={c}>baz</Component></div>;
+  return <div><div key="b">bar</div><div key={c}>baz</div></div>;
```

Comment:
the script has no idea about the value of `p` and `r`. This one should not be inlined.